src/interfaces/jira.ts

```diff
@@ -120,7 +120,13 @@ export interface JiraAuthor {
 	name: string;
 	url?: string;
 }
-export interface JiraReview extends JiraAuthor {
+
+export interface JiraReviewer {
```

Comment:
Adding a new interface here. We were using the same JiraAuthor before but `name`, `url` and `avatar` fields have been deprecated on the devinfo API
<img width="595" alt="Screenshot 2023-01-24 at 1 27 02 pm" src="https://user-images.githubusercontent.com/58801956/214200292-e684fdc6-1cf2-4590-a0c5-b7b4063e59fd.png">